src/dotnet/commands/dotnet-help/HelpCommandParser.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.DotNet.Cli.CommandLine;
+
+namespace Microsoft.DotNet.Tools.Help
+{
+    internal static class HelpCommandParser
+    {
+        public static Command Help()
+        {
+            return Create.Command(
+                "help",
+                LocalizableStrings.AppFullName,
+                Accept.ZeroOrOneArgument()
+                    .With(
+                        LocalizableStrings.CommandArgumentDescription,
+                        LocalizableStrings.CommandArgumentName),
+                Create.Option(
+                    "-h|--help|-?|/?",
```

Comment:
I find the problem. The test are testing _dotnet --help_ instead of _dotnet help --help_ . I should just remove this. And today, if you call _dotnet help -- help_ there will be an error thrown.